### ISSUE TITLE
Irritating "no metadata found" warning

### ISSUE THREAD

**Issue by MichalLytek** [typestack/class-validator#279]
> No metadata found. There is more than once class-validator version installed probably. You need to flatten your dependencies.

This warn is really irritating. In frameworks like routing-controllers or type-graphql, where `validate()` is called automatically, it's annoying to see that warning when someone hasn't used decorators for validation yet.

We should expose some option to disable that warning or sth like that.

**Comment by diegolaciar** [comment]
+1

**Comment by overpod** [comment]
A problem was detected when upgrading from 0.9.1 to 0.10.0

**Comment by sebs** [comment]
Is there a way to disable the debug message?

**Comment by lucas-wade** [comment]
I was having this issue but as soon as a i added a validator the error was gone.

**Comment by adams-family** [comment]
I'm having the same issue, here is my repro code:

```
import {validate} from "class-validator";

class Person {
  name: string;
  age: number;
}

var personObj = new Person();

validate(personObj).then(errors => {
  if (errors.length > 0) {
      console.log("validation failed. errors: ", errors);
  } else {
      console.log("validation succeed");
  }
});
```

Output:
```
No metadata found. There is more than once class-validator version installed probably. You need to flatten your dependencies.
validation succeed
```

I verified that I have only one version of `class-validator` installed:

```
find -name "class-validator*"
./node_modules/class-validator
```

**Comment by mersanuzun** [comment]
I'm having the same issue

"class-validator": "0.11.0"

**Comment by NoNameProvided** [comment]
Fixed in 68ed35d7b96348cd060dd86878773551c38c2b3f.
